### PR TITLE
Spin in button doesn't look good (fix #414)

### LINKED ISSUE #414: Spin in button doesn't look good
``` js
{
    block: 'button',
    mods: { theme: 'normal', size: 's' },
    content: [
        { block: 'spin', mods : { theme : 'normal', size : 'xs', progress : true } },
        { elem: 'text', content: 'Loading...' }
    ]
}
```

In result we have this:
http://yadi.sk/d/Zc4JPLvrMvAzj

### PR DESCRIPTION
**Pull request by sipayRT**

### CODE REVIEW

**Comment by mishanga** [comment]
Can you please append this example to [normal](../blob/v2/desktop.pages/normal/normal.bemjson.js) page?

**Comment by sipayRT** [comment]
yes, but in case spin+button we have button height difference:
button without spin - height: 24px
button+spin - height: 26px
it's all because of borders

**Comment by mishanga** [comment]
Borders will be fixed in #404

**Comment by persidskiy** [comment]
I resolved horizontal align of icons and spins in button with `button__icon` element https://github.com/bem/bem-components/pull/416

File: desktop.pages/normal/normal.bemjson.js [hunk 1]
```diff
@@ -141,6 +141,21 @@
                 },
                 ' ',
                 {
+                    block : 'button',
+                    mods : { theme : 'normal', size : 's' },
+                    content : [
+                        {
+                            block : 'spin',
+                            mods : { theme : 'normal', size : 's', progress : true }
```
Comment (mishanga) [inline desktop.pages/normal/normal.bemjson.js]: Надо использовать на размер меньше.

**Pull request merged**